Sources/TokamakCore/Views/TupleView.swift

```diff
@@ -22,65 +22,66 @@ public struct TupleView<T>: View {
 
   public let value: T
 
-  // FIXME: should not be public
-  public let children: [AnyView]
+  let _children: [AnyView]
 
   public init(_ value: T) {
     self.value = value
-    children = []
+    _children = []
   }
 
   public init(_ value: T, children: [AnyView]) {
     self.value = value
-    self.children = children
+    _children = children
   }
 
   init<T1: View, T2: View>(_ v1: T1, _ v2: T2) where T == (T1, T2) {
     value = (v1, v2)
-    children = [AnyView(v1), AnyView(v2)]
+    _children = [AnyView(v1), AnyView(v2)]
   }
 
+  // swiftlint:disable large_tuple
```

Comment:
<!--
  1 failure:  Tuples should hav...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/TupleView.swift;
  Line: 42;
-->
- :no_entry_sign: Tuples should have at most 2 members. (`large_tuple`)